### ISSUE TITLE
ts-evt -> evt ? 

### ISSUE THREAD

**Issue by garronej** [yelo-npm/Evt#1]
Hello  👋 

I am actively working on a type-safe replacement for ``EventEmitter`` targeting Deno but also Node and browsers. 
It is quite advanced, it got operators and chaining like RxJS.
Would you consider giving up the "evt" npm package name? 
It would be really clean to be able to have the same "evt" package name on Deno and Node.
Also, the domain name evt.land is available which would go along great with deno.land

This is the website of the project: [ts-evt.dev](https://ts-evt.dev)

Thank you for considering it. 🙏

**Comment by Xananax** [comment]
Yes, no problem, this package is abandoned anyway. It would be with pleasure! Can you send me your npm email to jad [at] mutnt.io?

I've perused the project, and it seems really cool. I'll probably end up using it. Kudos!
